### PR TITLE
Improvement to the MSMQ transport page

### PR DESCRIPTION
**Pull request by indualagarsamy**
Fixes https://github.com/Particular/docs.particular.net/issues/3334

- Added configuration snippet
- Added advantages and disadvantages of the transport.

### CODE REVIEW

**Comment by indualagarsamy** [comment]
@Particular/msmq-transport-maintainers - please review.

**Comment by indualagarsamy** [comment]
Looks like I will have to add partials for the different versions.

**Comment by janovesk** [comment]
Looks good @indualagarsamy!

File: transports/msmq/index.md [hunk 1]
```diff
@@ -2,16 +2,42 @@
...
+ - Transport
 ---
 
+## Configuring the endpoint
+
 partial: default
 
 
-## NServiceBus Configuration
+## Advantages and Disadvantages
```
Comment (andreasohlund) [inline transports/msmq/index.md]: Perhaps just kill this headline and move the sub ones one level up?

File: transports/msmq/index.md [hunk 2]
```diff
@@ -2,16 +2,42 @@
...
+
+
+### Advantages
+
+ * MSMQ is natively available as part of the Windows operating system. In Servers, the MSMQ role might need to be turned on.  
+ * MSMQ offers transactional queues which also support distributed transactions. With the transactional behavior, it is easy to get once-only delivery achievable.
```
Comment (andreasohlund) [inline transports/msmq/index.md]: >once-only

Think the accepted term is "exactly once"

Comment (janovesk) [reply transports/msmq/index.md]: Nope. It's `exactly-once` :)

File: transports/msmq/index.md [hunk 3]
```diff
@@ -2,16 +2,42 @@
...
+ * MSMQ provides a store and forward mechanism. Therefore it promotes a more natural bus-style architecture. When sending messages to unavailable servers, the messages are stored in the Outgoing queues and will be automatically delivered once the machine comes back online.
```
Comment (andreasohlund) [inline transports/msmq/index.md]: Link to https://docs.particular.net/transports/#types-of-transports-bus-transports ?

**Comment by indualagarsamy** [comment]
@andreasohlund - I have incorporated your suggestions.

**Pull request merged**